curriculum/challenges/english/01-responsive-web-design/basic-css-cafe-menu/part-046.md

```diff
@@ -8,14 +8,7 @@ isHidden: true
 ## Description
 <section id='description'>
 
-Within each of the four `article` elements, add the following desserts and prices using an `article` element with two nested `p` elements inside each. Similar to what you did with the coffees, the first `p` element's text should contain the dessert name and the second `p` element's text should contain the price.
-
-```bash
-Donut 1.50
-Cherry Pie 2.75
-Cheesecake 3.00
-Cinammon Roll
-```
+To complete the styling, add the applicable class names `flavor` and `price` to all the remaining `p` elements.
```

Comment:
To complete the styling, add the class names `flavor` and `price` to all the remaining `p` elements.

Reply:
The reason I had the original wording instead of what you propose, is I did not want the user to think they should add both class names to all the remaining `p` elements.  Any suggestion on how to accomplish that?

Reply:
@RandellDawson, ah, I see what you mean. I was more focused on simplifying, but I think you're right -- the original wording makes it more clear that the class names `flavor` and `price` should only be added to certain paragraph elements.

I don't think there's a better way to say that, so let's leave it as is.